manifests/prometheus.yml

```diff
@@ -0,0 +1,116 @@
+name: prometheus
+instance_groups:
+- name: alertmanager
+  azs: [ z1 ]
+  instances: 1
+  vm_type: default
+  persistent_disk_type: 1GB
+  stemcell: default
+  networks:
+  - name: default
+  jobs:
+    - name: alertmanager
+      release: prometheus
+      properties:
+        alertmanager:
+          route:
+            receiver: default-receiver
+          receivers:
+            - name: default-receiver
+
+- name: prometheus
+  azs: [ z1 ]
+  instances: 1
+  vm_type: default
+  persistent_disk_type: 1GB
+  stemcell: default
+  networks:
+  - name: default
+  jobs:
+  - name: prometheus_alerts
+    release: prometheus
+  - name: prometheus
+    release: prometheus
+    properties:
+      prometheus:
+        rule_files:
+        - /var/vcap/packages/prometheus_alerts/*.alerts
+        scrape_configs:
+        - job_name: prometheus
+          static_configs:
+          - targets:
+            - localhost:9090
+
+- name: grafana
+  azs: [ z1 ]
+  instances: 1
+  vm_type: default
+  persistent_disk_type: 1GB
+  stemcell: default
+  networks:
+  - name: default
+  jobs:
+  - name: prometheus_dashboards
+    release: prometheus
+  - name: grafana
+    release: prometheus
+    properties:
+      grafana:
+        auth:
+          basic:
+            enabled: true
+          disable_login_form: true
+        security:
+          admin_password: ((grafana_password))
+        dashboards:
+          json:
+            enabled: true
+        prometheus:
+          dashboard_files:
+          - /var/vcap/packages/prometheus_dashboards/*.json
+
+- name: nginx
+  azs: [ z1 ]
+  instances: 1
+  vm_type: default
+  stemcell: default
+  networks:
+  - name: default
+  jobs:
+    - name: nginx
+      release: prometheus
+      properties:
+        nginx:
+          grafana:
```

Comment:
We have 2 levels of authentication for grafana. Can we delete the `grafana` section from the `nginx` job please? Grafana will provide its own authentication method (that might be configured to use UAA for example).